chef/automate
# Update Studio helpers API endpoints

### :nut_and_bolt: Description: What code changed, and why?

In preparation for the GA Release of IAM v2, we're updating the API path from `/api/iam/v2beta` to `/api/iam/v2`

This change is only for the studio helpers. 

This change is block by the main PR https://github.com/chef/automate/pull/2533

### :chains: Related Resources
#2518 

### :+1: Definition of Done

All the hab studio helpers will use the new API path

### :athletic_shoe: How to Build and Test the Change
1. Start up Automate in the studio with `start_all_services`
1. Run `get_api_token_with_policy ingest-access` Ensure a token is returned
1. Run `remove_legacy_v1_policies` Ensure no errors are displayed

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
